src/cs2_sdk/entity/cbaseentity.h

```diff
@@ -135,9 +138,9 @@ class Z_CBaseEntity : public CBaseEntity
 	Vector GetAbsOrigin() { return m_CBodyComponent->m_pSceneNode->m_vecAbsOrigin; }
 	void SetAbsOrigin(Vector vecOrigin) { m_CBodyComponent->m_pSceneNode->m_vecAbsOrigin = vecOrigin; }
 
-	void Teleport(Vector *position, QAngle *angles, Vector *velocity) { CALL_VIRTUAL(void, offsets::Teleport, this, position, angles, velocity); }
+	void Teleport(Vector *position, QAngle *angles, Vector *velocity) { CALL_VIRTUAL(void, g_GameConfig->GetOffset("Teleport"), this, position, angles, velocity); }
 
-	void CollisionRulesChanged() { CALL_VIRTUAL(void, offsets::CollisionRulesChanged, this); }
+	void CollisionRulesChanged() { CALL_VIRTUAL(void, g_GameConfig->GetOffset("CollisionRulesChanged"), this); }
```

Comment:
GetOffset should only be run once here.

Reply:
its ok to keep it that way because we dont reload the file but just find the offset in a basic array